frontend/src/js/controllers/profileCtrl.js

```diff
@@ -123,6 +123,30 @@
             });
         };
 
+        // Disable User Account
+        vm.disableUserAccount = function() {
+            parameters.token = userKey;
+            parameters.method = 'POST';
+            parameters.url = 'accounts/user/disable';
+            parameters.callback = {
+                onSuccess: function(response) {
+                    var status = response.status;
+                    if (status == 200) {
+                        utilities.resetStorage();
```

Comment:
What does this exactly do here?

Reply:
it cleans the local storage/cookies to log out the disable user. without it my browser was stuck on unexceptional error trying to access the disable account.

Reply:
are we storing anything else in local storage in other places in the code?

Reply:
we are storing the userkey, jwt token and profile image, other than this mostly errors using utilities.storeData